src/otaclient/boot_control/_grub.py

```diff
@@ -741,6 +741,7 @@ def grub_reboot_to_standby(self):
 class GrubController(BootControllerProtocol):
     def __init__(self) -> None:
         try:
+            self._update_version = "unknown"
```

Comment:
How about to define "unknown" in `BootControllerProtocol` class and reuse it in each sub-class?